gpt_engineer/steps.py

```diff
@@ -280,15 +295,17 @@ class Config(str, Enum):
 # Different configs of what steps to run
 STEPS = {
     Config.DEFAULT: [
+        archive,
```

Comment:
Let's not make archive a step in this sense, but just call it before we loop over steps!